database/seeders/AccessibilitiesSeederAccounting.php

```
@@ -458,6 +458,13 @@ public function run(): void
                     'created_at' => Carbon::now(),
                     'updated_at' => Carbon::now(),
                 ],
+                [
+                    'id' => 1258,
+                    'accessibilities_name' => AccessibilityAccounting::ACCOUNTING_SETUP_REPORTGROUP->value,
+                    'deleted_at' => null,
+                    'created_at' => Carbon::now(),
```

PHPMD (2.15.0)

Avoid using static access to class '\Carbon\Carbon' in method 'run'. (undefined)

(StaticAccess)